src/main/java/com/laborhelper/enterprise/LaborHelperApplication.java

```diff
@@ -1,4 +1,7 @@
 package com.laborhelper.enterprise;
+/**
+*The use of @SpringBootApplication runs everything that @configuration, @EnableAutoConfiguration, and @ComponentScan with the default attributes
+*/
```

Comment:
1) This belongs above the class.
2) Most of this is a given to those who understand Spring Boot... so the comment isn't adding much value.